demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -1298,14 +1298,8 @@ def _collect_pack_items(pack_id: str, id_set: dict) -> dict:
                                      ('generic_definitions', 'GenericDefinitions'),
                                      ('lists', 'Lists'),
                                      ('jobs', 'Jobs')):
-            if id_set_key not in id_set:
-                raise RuntimeError(
-                    "\n".join((f"Error: the {id_set_key} content type is missing from the id_set.",
-                               "This may mean the existing id_set was created with an outdated version "
-                               "of the Demisto SDK. Please delete content/Tests/id_set.json and "
-                               "run demisto-sdk find-dependencies again."))
-                )
-            pack_items[pack_key] = PackDependencies._search_for_pack_items(pack_id, id_set[id_set_key])
+
+            pack_items[pack_key] = PackDependencies._search_for_pack_items(pack_id, id_set.get(id_set_key, []))
```

Comment:
why did you remove it?

Reply:
The error handling? since the id set inserted might be the marketplacev2 one, and it will have the missing items missing as keys, such as Dashboards, Layouts, etc

Reply:
we extract those as empty lists (https://github.com/demisto/demisto-sdk/pull/1745/commits/0a19d8e4d5e72051cfaf702166c85dbc6a84809a)

Reply:
True dat